src/Workspaces/CoreTest/ChecksumTests.cs

```diff
@@ -40,22 +32,18 @@ public void ValidateChecksumFromSpanSameAsChecksumFromBytes2()
             var checksum2 = Checksum.Create("Bar");
             var checksum3 = Checksum.Create("Baz");
 
-            var checksumA = Checksum.TestAccessor.CreateUsingByteArrays(checksum1, checksum2, checksum3);
-            var checksumB = Checksum.TestAccessor.CreateUsingSpans(checksum1, checksum2, checksum3);
+            var checksumA = Checksum.Create(checksum1, checksum2, checksum3);
 
-            Assert.Equal(checksumA, checksumB);
+            // Running this test on multiple target frameworks with the same expectation ensures the results match
+            Assert.Equal(Checksum.FromBase64String("NEfIznmqkIqi4VJl12KxycWt7uo="), checksumA);
 
             Assert.NotEqual(checksum1, checksum2);
             Assert.NotEqual(checksum2, checksum3);
             Assert.NotEqual(checksum3, checksum1);
 
             Assert.NotEqual(checksum1, checksumA);
-            Assert.NotEqual(checksum1, checksumB);
             Assert.NotEqual(checksum2, checksumA);
-            Assert.NotEqual(checksum2, checksumB);
             Assert.NotEqual(checksum3, checksumA);
-            Assert.NotEqual(checksum3, checksumB);
         }
-#endif
```

Comment:
i'm not a fan of this change here.  validating that the byte-array/span versions aren't checked against each other.

Reply:
Restoring this test to its original form would require additional conditional code that in practice defeats one of the key purposes of this test.

Reply:
Specifically, we need to _avoid_ using the `IncrementalHash` type on .NET Framework (due to the finalizers), but we need to _keep_ using the `IncrementalHash` type for incremental hashing on .NET Core because it provides `AppendData(ReadOnlySpan<byte>)` which is not present on the `SHA256` type.

Reply:
i see now.  thanks!